.github/workflows/wheels.yml

```diff
@@ -35,19 +34,32 @@ jobs:
         if:  startsWith(matrix.os, 'ubuntu')
         name: Set up QEMU
 
+      - name: Install prerequisites for macOS
+        if: contains(matrix.os, 'macos')
+        env:
+          # Best compatibility, even with older releases of macOS.
+          MACOSX_DEPLOYMENT_TARGET: "10.9"
+        run: |
+          brew install --build-from-source --no-binaries --force szip
+          brew reinstall --build-from-source --no-binaries --force c-blosc bzip2 hdf5 lz4 lzo snappy zstd zlib
+          brew link --overwrite --force bzip2 zlib
+
```

Comment:
Personally I would feel more comfortable with a `conda` based solution, but OK, I will merge this.

Reply:
i hope there is no misunderstanding here - you should not feel forced to merge PR's you're not comfortable with.

If there is a reason you think using brew is worse than using conda let me know, it's a PR for a reason (allowing discussions, ...) :)

Reply:
No it is just a personal feeling.
I know conda better than brew so I would prefer to go with conda but this is irrelevant for this PR IMHO.